3p/integration.js

```diff
@@ -126,6 +126,7 @@ window.draw3p = function(opt_configCallback) {
 
   // This only actually works for ads.
   window.context.observeIntersection = observeIntersection;
+  window.context.observeResize = observeResize;
```

Comment:
Been thinking for a while and I think it is better to just make this two functions
`onResizeSuccess` and `onResizeDenied`